data/korath/korath missions.txt

```diff
@@ -257,3 +257,566 @@ mission "Small Scale Delivery"
+			# Names:
+			# Pilot to the left: Akafavah, meaning "steel eyes"
+			# Pilot to the right: Tak Ratabomlek, or Takra for short, means "unforgotten"
+
+			# I don't know what "Oi" means.
+			choice
+				`	"I am <first> <last>."`
+					goto name
+				`	"Oi, oi!"`
+					goto silence
+				`	"I come in peace. Take me to your leader."`
+					goto peaceleader
+				`	(Remain silent and observe.)`
+					goto silence
+				`	(Attack them.)`
+					goto attack
+
+			label name
+			`	"Takra," it says, pointing to itself. Then it points to the other and says, "Akafavah."`
+			`	"Korath," it says, gesturing to the two of them. Then it looks at you expectantly.`
+				goto humanika
+
+			# With a longer starting sentence, they suspect you're speaking human standard
+			# (even if you may not be human) so they try to use the same language in response.
+			label peaceleader
+			`	Both of them seem confused, like they half-understand parts of what you said, but can't put the pieces together.`
+			`	"Peeese? Takra," it says, pointing to itself. Then it points to the other and says, "Akafavah." You respond almost instinctively with "<first> <last>."`
+			`	"Weee Korath," it says, gesturing to the two of them. Then it gestures at you and says, "Yee ess...?"`
+				goto humanika
+
+			label humanika
+			`	"Human," you say, almost instinctively.`
+			`	Akafavah says, "Oy! Humanika!" It's not clear what emotion they're feeling, but it's a strong one.`
+			`	Takra asks you, "Were foam?"`
+			`	They see your confusion and seem to give up on communicating in your language. It's time for a different approach.`
+				goto map
+
+			label silence
+			`	The left gestures at itself and says, "Akafavah." The right does a similar gesture and says, "Takra." You respond almost instinctively with "<first> <last>."`
+			`	Pleased with this success, Akafavah continues, "Korath," gesturing at the two pilots.`
+			`	"I'm a human," you say, "from Earth."`
+			`	Akafavah says suddenly, "Oi! Humanika!" It's unclear what emotion the Korath is expressing, but it is a strong one.`
+			`	They see your confusion and seem to give up on communicating with words. It's time for a different approach.`
+				goto map
+
+			label attack
+			`	You reach for your gun. They move faster than you would have expected possible. One of strikes a pose with their arms and shouts, "Nag! Nag! Koreshi far'en lai!" Simultaneously, the other dives at you, grabbing the gun out of your hands before you have a chance to fire it.`
+			`	Then, instead of attacking you, the one who took your gun begins waving it your face and talking very quickly, as if scolding you. It points to the sky with a sweeping gesture, then mimics drawing the gun as you just did. Then it points to the ground, puts the gun on the ground, and says, "Far'en lai ko forei lak'far." Then it picks up the gun and repeats both gestures: all the sky, gun drawn. Earth, gun on ground. Then it tosses the gun down at your feet and waits, expectantly.`
```

Comment:
I want @ravenshining's opinion on changing this one.

Reply:
I think "down" makes the most sense.

Then it picks up the gun and repeats both gestures: all the sky, gun drawn. **Down,** gun on ground. Then it tosses the gun **near your feet** and waits, expectantly.

Reply:
This is a minor gripe, and I think everyone will be okay with it, so I'll resolve this conversation to avoid cluttering the gigantic comment chain. Please comment again if there are problems with the new text.